NuRadioReco/detector/detector.py

```diff
@@ -33,11 +33,7 @@ def decode(self, s):
         return datetime.strptime(s, '%Y-%m-%dT%H:%M:%S')
 
 
-serialization = SerializationMiddleware()
-serialization.register_serializer(DateTimeSerializer(), 'TinyDate')
-
-
-def buffer_db(in_memory, filename=None):
+def buffer_db(in_memory, filename=None, serialization=None):
```

Comment:
Can more docs be added on what these arguments do? In particular, the purpose of `in_memory` and `serialization`? I don't see where the `serialization` argument would be used except in the case where `in_memory=False`? SO just a bit more clarity there would be nice. Thanks!